packages/Link/Link.modules.scss

```diff
@@ -30,7 +30,7 @@
 
   &:link,
   &:visited {
-    color: $color-link;
+    color: #2a2c2e;
```

Comment:
I guess we had some miscommunication here, sorry about that. We should not be hardcoding the color like this and should be using a variable from the colour palatte, just not `color-link`

I think what we want is `color-shark` or `color-text`. Maybe still using `color-text` is fine because it needs to be shared by multiple components, Headings, Link, Text, etc.

Reply:
Haha, yeah, just realized that could be the case. I'll change it to `color-shark` to make it more resilient to future changes to `core-colour`.